### ISSUE TITLE
identifierForVendor change when reset 

### ISSUE THREAD

**Issue by hoangde2811** [fluttercommunity/plus_plugins#775]
## System info

Platform the Issue occurs on:
Plugin name:
Plugin version:

## Steps to Reproduce

1. ...
2. ...
3. ...

<details>
<summary>Logs</summary>

```dart
```

</details>

<details>
<summary>Flutter doctor output</summary>

```
```

</details>

When resetting iphone, identifierForVendor is changed
When resetting iphone, identifierForVendor is changed, please developers check again.
Sincerely thank you.

**Comment by Leffe108** [comment]
Well, if you lookup the [Apple docs](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor), down the bottom it says that the ID gets re-generated when a user uninstalls all apps published by you in the App Store. It doesn't specifically say it but, I believe a vendor is identified by the team-ID in AppStore. 

So I would say this is not a bug, but just is how identifierForVendor works on iOS.

IMHO I think it is good that this plugin doesn't try to persist the id. I found a plugin for cordova that did that and I had to re-create the plugin myself without persistence as the way they did it could end up on the backup and thus restored on a different device.
